llm_studio/src/datasets/text_causal_regression_ds.py

```diff
@@ -28,11 +28,28 @@ def __getitem__(self, idx: int) -> Dict:
         return sample
 
     def postprocess_output(self, cfg, df: pd.DataFrame, output: Dict) -> Dict:
-        output["logits"] = output["logits"].float()
-        preds = output["logits"]
-        preds = np.array(preds).astype(float).astype(str).reshape(-1)
+        output["predictions"] = output["predictions"].float()
+        preds = []
+        for col in np.arange(len(cfg.dataset.answer_column)):
+            preds.append(
+                np.round(output["predictions"][:, col].cpu().numpy(), 3).astype(str)
```

Comment:
specific reason for the rounding here?

Reply:
yeah this is shown in the visualizations and with lots of digits not great to read

Reply:
Then, why not truncate in the visualization? The current implementation also rounds the downloadable predictions.

Reply:
It doesnt because `output["predicted_text"]` is not used there

Reply:
Ah, right, that was changed in the PR. Still is odd for the exported dataframe to have rounded values

Reply:
Hmh, debatable. Would be tricky to add something different there. The pickle should be used anyways for exact values.